src/transformers/utils/versions.py

```diff
@@ -42,9 +42,9 @@
 
 def _compare_versions(op, got_ver, want_ver, requirement, pkg, hint):
     if got_ver is None:
-        raise ValueError("got_ver is None")
+        raise ValueError(f"got_ver is None when checking {requirement} for {pkg}")
     if want_ver is None:
-        raise ValueError("want_ver is None")
+        raise ValueError(f"want_ver is None when checking {requirement} for {pkg}")
```

Comment:
I'll make the suggested change and push it up shortly.

I somehow ended up with empty packages installed. As in, the folders and the egg info directory exist, so the package look like it's installed to importlib but then it has no version information (and no files to actually load). It's quite an invalid state but I have _no idea_ how it got to be like that beyond the possibility of arbitrary file _deletion_: our venv was on a shared system and a junior admin haphazardly deleted all files with creation dates older than 2022-01-01 to free up space… this created a lot of weirdness! However, I'd reinstalled the venv and its packages with Conda _after_ that event, so I'm _really_ not sure what happened. Manually installing the packages with pip brought 'em back. Assumptions about shared filesystems made by my predecessors were… unsafe. And transformers is getting slightly better because of it!

Reply:
Thank you for sharing the details of your use-case. This is definitely a possibility for users, so yes, I'm glad you are making the Transformers code more user-friendly, especially so when it's already a troublesome situation.

I'm not sure if perhaps it'd help to also add at the end: `"you may want to try to force the reinstall of the {pkg}"`

Reply:
@colindean, currently the autoformatter CI job fails - you need to run `make style` and push the changes. run `pip install -e .[dev]` if the former fails.

Reply:
I wasn't able to get `pip install -e .[dev]` to complete successfully.

```
ERROR: Could not find a version that satisfies the requirement tensorflow-text; extra == "dev" (from transformers[dev]) (from versions: none)
ERROR: No matching distribution found for tensorflow-text; extra == "dev"
```

I'm on an M1 Mac and there's no whl for macos arm64 for `tensorflow-text` in Tensorflow 2.9.0. Also, fugasi required installation of `mecab` from Homebrew… I'll see about adding some hints to CONTRIBUTING about that last one but I don't have a good workaround for the former. Its unavailability blocks installation of packages :-/